xee/ext.py

```diff
@@ -195,6 +203,12 @@ def __init__(
       executor_kwargs = {}
     self.executor_kwargs = executor_kwargs
 
+    self.tile_fetch_kwargs = (
```

Comment:
`self.tile_fetch_kwargs = {**self.TILE_FETCH_KWARGS, **(tile_fetch_kwargs or {})}`

The `tile_fetch_kwargs` provided by users may be incomplete.